online-shop/src/order.ts

```diff
@@ -0,0 +1,11 @@
+import { OrderDetails } from './order-details';
+
+export interface Order {
+  id: number;
```

Comment:
id can be optional, because when you send an order from UI, you don't have an id